README.rst

```diff
@@ -80,16 +120,22 @@ This section outlines what stages a proposal may go through. The stage is identi
    * `How to comment on a proposal <#how-to-comment-on-a-proposal>`__
    * `≡ List of proposals under discussion <https://github.com/ghc-proposals/ghc-proposals/pulls?q=is%3Aopen+is%3Apr+no%3Alabel>`__
  
-   At some point the proposal author brings the proposal before the committee for review, which will prompt the committee secretary to appoint a shepherd for the proposal.
+4. (No label.)  **Responsibility for next action:** author.
+
+   *Note: The Reviewing process does NOT start automatically when the discussion ends.*
+
+   At some point, the proposal author wishes to start a review process of the proposal by the committee. 
+
+   Then the author MUST explicitly ask in comments the committee secretary to bring the proposal before the committee for review (and to appoint a shepherd for it).
 
    * `How to bring a proposal before the committee <#how-to-bring-a-proposal-before-the-committee>`__
    * `Who is the committee? <#who-is-the-committee>`__
 
-4. (No label.)  **Responsibility for next action:** committee secretary.  **Timescale**: a few days.
+5. (No label.)  **Responsibility for next action:** committee secretary.  **Timescale**: a few days.
 
    The committee secretary appoints a committee member as shepherd, which moves the proposal to the *Pending shepherd recommendation* state.
 
-5. **Label**: `Pending shepherd recommendation <https://github.com/ghc-proposals/ghc-proposals/pulls?q=is%3Aopen+is%3Apr+label%3A%22Pending+shepherd+recommendation%22>`_.  **Responsibility for next action:** shepherd.  **Timescale**: two weeks.
+6. **Label**: `Pending shepherd recommendation <https://github.com/ghc-proposals/ghc-proposals/pulls?q=is%3Aopen+is%3Apr+label%3A%22Pending+shepherd+recommendation%22>`_.  **Responsibility for next action:** shepherd.  **Timescale**: two weeks.
 
    The shepherd reviews the proposal, and may request changes that they would like to see before they submit their recommendation to accept or reject the proposal to the committee.
```

Comment:
While we're here, the next two paragraphs (not actually part of the diff) could be clearer that if the shepherd requests revisions, they are not also expected to submit a recommendation within two weeks. Perhaps we should say "Within two weeks, if the shepherd does not request revisions, then they should..."